shared-components/src/components/textfield/TextField.tsx

```diff
@@ -13,9 +13,11 @@ export const TextField = forwardRef<
     name: string;
     label: string;
     fullWidth?: boolean;
-    lightBackground?: boolean;
+    valid?: boolean;
+    lightBackground?: boolean | "white";
+    whiteBackground?: boolean;
     labelClass?: string;
-    error?: string | null | false;
+    errorMessage?: string | null | false;
```

Comment:
This is a breaking change. So then we should weight is this slight improvement worth updating every usage? It might, just needs to be considered.